data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -184,12 +184,23 @@
             id=Engineer
             name= _ "Engineer"
             unrenamable=yes
-            type=Red Mage
+            type={ON_DIFFICULTY "Arch Mage" "Silver Mage" "Red Mage"}
             x,y=18,5
             side=1
             random_traits=no
             [modifications]
                 {TRAIT_LOYAL}
+#ifdef EASY
```

Comment:
I'm not totally sure what to think about mr. engineer having extra hp. Maybe could be okay? Would help if you could elaborate a tiny bit on why you want this.

Reply:
I just had a really hard time keeping him alive; I'm open to replacing it with an additional trait or two instead if that'd make more sense

Reply:
I'd say traits seem better to me. Maybe resilient and another should you so choose. (Healthy just for fun?)

Reply:
ok I replaced it with those traits in b56b92c